app/controllers/v0/evss_claims_controller.rb

```diff
@@ -25,8 +26,9 @@ def show
     def request_decision
       claim = EVSSClaim.for_user(current_user).find_by(evss_id: params[:id])
       raise Common::Exceptions::RecordNotFound, params[:id] unless claim
+
       jid = service.request_decision(claim)
-      claim.update_attributes(requested_decision: true)
```

Comment:
can we revert all of the ` update_attributes` changes?  It would be a change in behavior that could possibly cause bugs so I'd rather not see it as part of a huge PR like this one.